src/main/java/com/zemnov/salon/service/ChequeSaveService.java

```diff
@@ -0,0 +1,30 @@
+package com.zemnov.salon.service;
+
+import org.springframework.stereotype.Service;
+
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+
+@Service
+public class ChequeSaveService {
```

Comment:
We should not operate with classes, but with interfaces. All those services must be interfaces and have  implementations